tests/tasks/setup/eks/awscli-cfn-lt-al2023.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cfn-launch-template-al2023
+  namespace: scalability
+spec:
+  description: |
+    Create an EKS CFN stack to output a launch template for AL2023-based nodes.
+    This Task can be used to create an EKS CFN stack that outputs a launch template.
+    The launch template may be used for a managed nodegroup with or without a custom AMI.
+  params:
+    - name: cluster-name
+      description: EKS cluster you want to create CFN stack for.
+    - name: stack-name
```

Comment:
I think we can avoid this param and leverage cluster-name ? Usually we try to pass as less params as possible in pipeline def. wdyt ?

Reply:
The intention was for this to be a drop-in replacement for the existing LT task, so if this is removed I think it should be done in a later PR